igor-core/src/main/java/com/netflix/spinnaker/igor/build/model/GenericArtifact.java

```diff
@@ -45,4 +47,5 @@ public GenericArtifact(String type, String name, String version, String referenc
   private String type;
   private String version;
   private Map<String, String> metadata;
+  private boolean decorated;
```

Comment:
This will serialize this field as well and surface it to the pipeline execution. I had originally annotated it with `@JsonIgnore` but I removed it again because I felt it wouldn't hurt to actually expose the field. I'm happy to re-add it if someone feels different.

Reply:
Actually, I found a use for the flag (in https://github.com/spinnaker/orca/pull/3422), so now I'm not so happy to ignore it anymore 😎